components/bear-market-airdrop/Blocks/Hero/ContractsDeployed.tsx

```diff
@@ -41,13 +37,11 @@ const ListItem: React.FC<ListItemProps> = ({ contract, walletAddress }) => {
         textDecoration: "none",
       }}
       role="group"
-      onClick={() => {
-        trackEvent({
-          category: bearMarketTrackerCategory,
-          action: "click",
-          label: "Contracts Deployed: Contract Link",
-          walletAddress,
-        });
+      category={bearMarketTrackerCategory}
+      trackingProps={{
+        action: "click",
+        label: "contracts_deployed",
```

Comment:
you can pass label as a prop in the main component, in this case is the same but if you don't have anything extra it's better so you don't need to have trackingProps